schema/cron.sh

```diff
@@ -30,19 +30,19 @@ if [ ! -z "$existing_branch" ]; then
 fi
 
 # Generate enigma-go based on latest published Qlik Associative Engine image
-./schema/generate.sh
+. ./schema/generate.sh
```

Comment:
?

Reply:
I need to source the generate script to be able to access the `ENGINE_VERSION` variable.